custom_components/adjustable_bed/support_report.py

```
@@ -0,0 +1,238 @@
+    # Get adapter info if available
+    try:
+        adapters = bluetooth.async_get_adapters(hass)
+        info["adapters"] = [
+            {
+                "address": adapter.get("address"),
+                "adapter_name": adapter.get("name"),
+                "passive_scan": adapter.get("passive_scan"),
+            }
+            for adapter in adapters
+        ]
+    except Exception as err:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)